docker-compose.yaml

```diff
@@ -8,6 +8,7 @@ services:
       target: base
     ports:
       - '4000:4000'
+      - '9229:9229'
```

Comment:
What is this port for? Is there a reason for choosing 9229?

Reply:
From nodejs documentation. Doesn't have any meaning for me
https://nodejs.org/en/docs/guides/debugging-getting-started/

Reply:
I see thanks I was just curious